src/components/footer.js

```diff
@@ -43,30 +44,30 @@ const Footer = (props) => {
         <div className="grid-item-3">
           <div className="content">
             <div className="column">
-              <a href="">COMPANY</a>
-              <a className="body-small" href="">
+              <Link to="">COMPANY</Link>
+              <Link className="body-small" to="">
                 About
-              </a>
-              <a className="body-small" href="">
+              </Link>
+              <Link className="body-small" to="">
                 Team
-              </a>
-              <a className="body-small" href="">
+              </Link>
+              <Link className="body-small" to="">
                 Press
-              </a>
+              </Link>
             </div>
-            <a href="/services">SERVICES</a>
+            <Link to="/services">SERVICES</Link>
             <div className="column">
-              <a href="">OUR WORK</a>
-              <a className="body-small" href="">
+              <Link to="">OUR WORK</Link>
+              <Link className="body-small" to="">
                 Case studies
-              </a>
-              <a className="body-small" href="">
+              </Link>
+              <Link className="body-small" to="">
                 Approach
-              </a>
+              </Link>
             </div>
-            <a href="">INSIGHTS</a>
-            <a href="">CARREERS</a>
-            <a href="">CONTACT</a>
+            <Link to="">INSIGHTS</Link>
+            <Link to="">CARREERS</Link>
```

Comment:
@Yodart I'm going to merge this as-is. It may be better to do the above as part of the Footer refactor ticket: https://civicactions-home.atlassian.net/browse/HOMER20-224